src/main/java/uk/ac/ebi/intact/app/internal/model/managers/Manager.java

```diff
@@ -8,8 +8,8 @@
 import java.util.Properties;
 
 public class Manager {
-    private static final String INTACT_WEBSERVICES = "https://www.ebi.ac.uk/intact/ws/";
-//    private static final String INTACT_WEBSERVICES = "http://127.0.0.1:8081/intact/ws/";
+//    private static final String INTACT_WEBSERVICES = "https://www.ebi.ac.uk/intact/ws/";
```

Comment:
We need to revert this to the main URL before merge. I will deploy the latest changes to prod soon and I will comment here when done.